tests/pluginMetricsRequests.spec.ts

```diff
@@ -0,0 +1,189 @@
+import * as chanceJs from 'chance';
+import { register } from 'prom-client';
+import { Application } from 'express';
+import { IBasicAuth, IStorageManager, PluginOptions } from '@verdaccio/types';
+
+import MetricsPlugin, { DEFAULT_METRICS_PATH } from '../src';
+import { MetricsConfig } from '../src/types';
+import { AuthType, UNKNOWN } from '../src/utils';
+
+// @ts-ignore
```

Comment:
Would like a comment on why this is necessary or if not necessary, we should remove it.